modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm

```diff
@@ -39,6 +39,19 @@
 	name = "high heels"
 	desc = "A fancy pair of high heels. Won't compensate for your below average height that much."
 	icon_state = "heels"
+	greyscale_config = /datum/greyscale_config/heels
+	greyscale_config_worn = /datum/greyscale_config/heels/worn
+	greyscale_config_worn_digi = /datum/greyscale_config/heels/worn/digi
+	greyscale_colors = "#FFFFFF"
+	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = TRUE
+
+/obj/item/clothing/shoes/black_heels
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
+	name = "black heels"
+	desc = "A pair of black high heels that are much smaller on your feet"
+	icon_state = "highheels"
```

Comment:
We typically always put `name` and `desc` first, and try to basically put the long variables lower than the short ones, when possible, to make easier to read through.
```suggestion
	name = "black heels"
	desc = "A pair of black high heels that are much smaller on your feet"
	icon_state = "highheels"
	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
```